scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,52 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def child_to_parent_unmap(cell_parent, gpio_index):
+    # This function returns a (gpio-controller, pin number) tuple from
+    # cell_parent (identified as a 'nexus node', ie: has a 'gpio-map'
+    # property) and gpio_index.
+    # Note: Nexus nodes and gpio-map property are described in the
+    # upcoming (presumably v0.3) Device Tree specification, chapter
+    # 'Nexus nodes and Specifier Mapping'.
+
+    # First, retrieve gpio-map as a list
+    gpio_map = reduced[cell_parent]['props']['gpio-map']
+
+    # Before parsing, we need to know raws size.
```

Comment:
Still says "raws". Could say "the size of each entry in 'gpio-map'" or the like.

Reply:
Ohh... maybe 'raws' is a typo for 'rows'. I kept reading it as something "raw".

"Before parsing, we need to know the row size in 'gpio-map'", maybe.

Reply:
Indeed, fixed